tests/docs/smoke-all/2023/06/13/issue-4619.qmd

```diff
@@ -0,0 +1,9 @@
+---
```

Comment:
Nice test.

Reply:
This test will really only check that the rendering doesn't error AFAIK (e.g. the test runner will look for either an error or warning during the rendering of the document). To truly verify this, you'd want to look at the generated CSS for the page, but we don't really have a great way to do that right now (and really I don't even think that we'd want to look at the rendered CSS, but instead 'see' that the external icon doesn't break, since that is really what we want to test).

I'd recommend that we remove the test for now - I don't think its adding much and will make the test runner a bit slower.